### DISCUSSION TITLE
`CODEOWNERS` 파일 추가

### DISCUSSION THREAD

**Discussion by scarf005** [jiphyeonjeon-42/frontend#543] [Ideas]
[`CODEOWNERS`파일](https://docs.github.com/ko/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners)을 써서 '담당자'를 지정하면 PR 리뷰어를 자동으로 지정할 수 있습니다.

1. 파일 경로 '담당자'를 지정합니다. 예시:

- `/src/ @jiphyeonjeon-42/tf`: src 경로 아래 모든 파일, 담당팀 `@jiphyeonjeon-42/tf`
- `*.css @scarf005 @notusing11`: 모든 css 파일, 담당자 `@scarf005`, `@notusing11`

2. 담당자를 지정하면 수정하는 PR이 열릴 때 자동으로 리뷰어로 할당됩니다.
3. 이 기능을 활용하면 코드 리뷰어 선정 프로세스를 자동화할수 있습니다.

**Comment by not-using** [comment]
팀원 추가 및 변경이 있을 때마다 CODEOWNERS 파일을 갱신해야 하는게 더 불편할 것 같습니다

**Reply by Jake1152** [reply]
안건을 제시하시게된 배경이 궁금합니다:)

**Reply by scarf005** [reply]
PR 리뷰어를 자동으로 지정하면 편할것 같아서입니다.

**Reply by Jake1152** [reply]
@scarf005 넵 답변 고맙습니다 ㅎ
